### PR TITLE
Exit with an error if a glob didn't expand to any file

### PR DESCRIPTION
**Pull request by verdie-g**

### CODE REVIEW

**Comment by verdie-g** [comment]
Fixes #130

File: XmlSchemaClassGenerator.Console/Program.cs [hunk 1]
```diff
@@ -90,15 +90,32 @@ static void Main(string[] args)
...
+                if (Uri.IsWellFormedUriString(globOrUri, UriKind.Absolute))
+                {
+                    uris.Add(globOrUri);
+                    continue;
+                }
+
+                var expandedGlob = Glob.ExpandNames(globOrUri).ToList();
+                if (expandedGlob.Count == 0)
+                {
+                    System.Console.WriteLine($"No matches for glob {globOrUri}");
```
Comment (mganss) [inline XmlSchemaClassGenerator.Console/Program.cs]: The error message should be more user-friendly: "No files found for ..." or similar.

**Comment by mganss** [comment]
The diff is very large now. Is this a problem with whitespace/newlines?

**Comment by verdie-g** [comment]
huh

**Comment by mganss** [comment]
If you click on "Files changed" the whole file is shown as changed, although it should be only a couple of lines. This could be a problem with different newlines (CR vs LF etc). I don't think it's spaces or tabs as these would be shown in the GitHub diff.

**Comment by verdie-g** [comment]
Yea, I saw that. My mistake. It's fixed.

**Pull request merged**